frontend/market/market.tsx

```diff
@@ -166,6 +165,6 @@ class _MarketComponent extends React.PureComponent<Props, State> {
 }
 
 export const MarketComponent = connect((state: RootState, ownProps: OwnProps) => ({
-    shouldShowPrelude: state.teamState.marketStatus == MarketStatus.Forced,
+    shouldShowPrelude: state.teamState.marketStatus == MarketStatus.Open,
```

Comment:
I think this change may make the "prelude" show every time the market is accessed? Because before I relied on the fact that we only force the market the first time.

Reply:
Oh I see it does, but maybe you intended that.

Reply:
Yeah I think it's fine now that the intro's really short + quick.